src/pages/Detail/DetailIntroduction.js

```diff
@@ -0,0 +1,286 @@
+import React from 'react';
+import styled from 'styled-components';
+
+function DetailIntroduction() {
+  return (
+    <IntroductionWrapper>
+      <IntroHeader>
+        <CategoryTitle>
+          <span>category</span>
+        </CategoryTitle>
+        <ProjectTitle>
+          <h1>유기동물 입양 문화를 만드는 포인핸드 매거진 8호</h1>
+        </ProjectTitle>
+        <CreatorProfile>
+          <img alt="profileImg" src="/images/example2.jpg" />
+          <p>creator name</p>
+        </CreatorProfile>
+      </IntroHeader>
+      <IntroImage alt="introImage" src="/images/example2.jpg" />
+      <IntroAside>
+        <ProjectStatus>
+          <CollectedAmount>
+            <p>모인금액</p>
+            <h2>3,505,000 원</h2>
+          </CollectedAmount>
+          <LeftDate>
+            <p>남은시간</p>
+            <h2>8일</h2>
+          </LeftDate>
+          <Sponsor>
+            <p>후원자</p>
+            <h2>64명</h2>
+          </Sponsor>
+        </ProjectStatus>
+        <ProjectButtons>
+          <PushThisProjectBtn>
+            <button>프로젝트 밀어주기</button>
+          </PushThisProjectBtn>
+          <LikeBtn>
+            <button>
+              <i className="far fa-heart"></i>
+            </button>
+          </LikeBtn>
+          <ShareBtn>
+            <button>
+              <i className="fas fa-share-alt"></i>
+            </button>
+          </ShareBtn>
+        </ProjectButtons>
+      </IntroAside>
+    </IntroductionWrapper>
+  );
+}
+
+const IntroductionWrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  padding-bottom: 1rem;
+  line-height: 1.7em;
+  word-break: break-all;
+
+  ${({ theme }) => theme.desktop`
+  justify-content:center;
+  `};
```

Comment:
```suggestion
  ${({ theme }) => theme.desktop`
    justify-content:center;
  `};
```
indent 맞춰져 있으면 좋을거 같아요
eslint, prettier 잘 작동하고 있나요?

Reply:
앗! 왜인지 styled component안에 prettier가 작동하지 않는것 같습니다! 확인하고 수정하겠습니다!!